src/parser.ts

```diff
@@ -584,13 +586,15 @@ function parseMultipleSwitchFromToolArguments(args: string, sw: string, removeSu
 
     match = regexp.exec(args);
     while (match) {
-        let matchIndex: number = (match[2].startsWith("'") && match[2].endsWith("'")) ? 8 : 18;
+        let matchIndex: number = (match[2].startsWith("'") && match[2].endsWith("'")) ? 8 : 21;
         let result: string = match[matchIndex];
         if (result) {
            if (removeSurroundingQuotes) {
               result = util.removeSurroundingQuotes(result);
            }
             results.push(result);
+        } else {
+           logger.message(`!!!Double check matchIndex!!! Remove in last commit befor merge!!! args = ${args}`);
```

Comment:
Remove this?

Reply:
I forgot to mark the PR as draft. I just wanted to get it out from one computer so that I git pull on others and work on tests. At the last commit I'll remove. I also need to see this message while running the tests that were already there.
I'm also working now on the last test relevant for this pattern change.